src/Containers.jsx

```diff
@@ -144,15 +144,15 @@ const ContainerActions = ({ container, containerDetail, healthcheck, onAddNotifi
             args.t = 0;
         client.postContainer(container.isSystem, "restart", container.Id, args)
                 .catch(ex => {
-                    const error = cockpit.format(_("Failed to restart container $0"), container.Names);
+                    const error = cockpit.format(_("Failed to restart container $0"), container.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)